react-electron-poc/src/renderer/components/panels/TextPanels/ScriptureTextPanelSlate.tsx

```diff
@@ -490,32 +455,21 @@ const withScrMarkers = (editor: CustomEditor): CustomEditor => {
                                         },
                                     );
                                 }
-
-                                console.log(
-                                    'After:',
-                                    JSON.stringify(editor.children, null, 2),
-                                );
-                                console.log(editor.selection);
                             }
                         } else {
                             // Closing an existing marker
                             // Get closest element of this marker style
                             const markerElement = Editor.above(editor, {
                                 match: (n) =>
                                     Element.isElement(n) &&
-                                    n.type !== 'editor' &&
+                                    // n.type !== 'editor' &&
```

Comment:
Super minor thing. Can this be removed? You might need it later so do whatever you think is best.

Reply:
Good call - I will remove it. Thank you!